doc/classes/@GDScript.xml

```diff
@@ -776,6 +776,16 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="randn">
+			<return type="float">
+			</return>
+			<description>
+				randn(m, s) returns a normally distributed random number with m as mean as s as standard deviation.
+				[codeblock]
+				randn(10.0, 5.0) # returns 7.23421
```

Comment:
Obligatory xkcd: 😃 
![](https://imgs.xkcd.com/comics/random_number.png)
(Yea, I know, it is the same for the randf documentation)

Reply:
http://dilbert.com/strip/2001-10-25